sources/packages/web/src/components/students/applicationTracker/generic/ApplicationStatusTrackerBanner.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <v-alert
+    variant="tonal"
+    class="application-tracker-alert mt-3"
+    :class="backgroundColor"
+    :color="backgroundColor"
+  >
+    <template #prepend>
+      <v-icon class="mt-1" :icon="icon" :size="20" :color="iconColor" />
+    </template>
+    <template #title>
+      <div class="category-header-medium mb-3 label">
+        {{ label }}
+      </div>
+    </template>
+    <span class="label-value-normal">
+      <slot name="content">
+        <v-row>
+          <v-col :md="hasImage ? 10 : 12">
```

Comment:
Just for clarification, without setting md will it not adjust based on the next column if it is available.?